aya/src/programs/tc.rs

```diff
@@ -228,6 +228,92 @@ define_link_wrapper!(
     TcLinkId
 );
 
+/// [`SchedClassifierLink`] is the link type used by [`SchedClassifier`] programs.
+///
+/// # Examples
+///
+/// ```no_run
+/// # #[derive(Debug, thiserror::Error)]
+/// # enum Error {
+/// #     #[error(transparent)]
+/// #     IO(#[from] std::io::Error),
+/// #     #[error(transparent)]
+/// #     Map(#[from] aya::maps::MapError),
+/// #     #[error(transparent)]
+/// #     Program(#[from] aya::programs::ProgramError),
+/// #     #[error(transparent)]
+/// #     Bpf(#[from] aya::BpfError)
+/// # }
+/// # let mut bpf = aya::Bpf::load(&[])?;
+/// # tc::qdisc_add_clsact("eth0")?;
+/// # let prog: &mut SchedClassifier = bpf.program_mut("redirect_ingress").unwrap().try_into()?;
+/// # prog.load()?;
+/// #
+/// # use aya::programs::tc::{SchedClassifierLink};
+/// # use aya::programs::{tc, Link, SchedClassifier, TcAttachType};
```

Comment:
When re-doing the example, please move all the imports to the top. You can also merge those two lines above into one:

```suggestion
/// # use aya::programs::{tc::{self, SchedClassifierLink}, Link, SchedClassifier, TcAttachType};
```